app/scripts/scripts.js

```diff
@@ -1,17 +1,28 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
+(function () {
```

Comment:
Added scripts.js to the bottom of index.html. My files got mixed up with Marcus's commit for some reason, but that's been fixed.